### PR TITLE
Add basic infrastrucutre and model changes for ReqIF requirements import

### PR DESCRIPTION
**Pull request by franzTobiasDLR**
This PR adds RMF resource and some model changes that will be necessary to import requirements.

PR #950 and #947  should me merged first. Their changes are contained here.

### CODE REVIEW

File: de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/wizard/ReqIfMappingPage.java [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+		table.setHeaderVisible(true);
+		table.addSelectionListener(this);
+
+		TableColumn columnColumnName = new TableColumn(table, SWT.NULL);
+		columnColumnName.setWidth(COLUMN_NAME_WIDTH);
+		columnColumnName.setText(COLUMN_SPECIFICATION_LABEL);
+
+		TableColumn columnType = new TableColumn(table, SWT.NONE | SWT.DROP_DOWN);
+		columnType.setText(COLUMN_CONTAINER_LABEL);
+		columnType.setWidth(COLUMN_TYPE_WIDTH);
```
Comment (JAmmermann-DLR) [inline de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/wizard/ReqIfMappingPage.java:90]: Maybe a more spelling name, e.g. `columnSpecification` and `columnContainer` ? Then also rename the width constants?

File: de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/wizard/ReqIfMappingPage.java [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+		EList<Specification> specList = reqIFImpl.getCoreContent().getSpecifications();
+		for (Specification spec : specList) {
+			TableItem item;
+			int index = specList.indexOf(spec);
+			if (tableItems.size() <= index) {
+				item = new TableItem(table, SWT.NULL);
+				tableItems.add(item);
+			} else {
+				item = tableItems.get(index);
+			}
```
Comment (JAmmermann-DLR) [inline de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/wizard/ReqIfMappingPage.java:112]: Here we override the existing items. We miss the case if we select a file with many specifications, go back and select a file with less specifications:
grafik
->
grafik

So we also have to remove unused items here.

File: de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/wizard/ReqIfMappingPage.java [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+	/**
+	 * Get the mapping of specifications on their container
+	 * @return the map of specs on seis
+	 */
+	public Map<Specification, StructuralElementInstance> getSpecificationMapping() {
+		Map<Specification, StructuralElementInstance> mapSeiToSpec = new HashMap<Specification, StructuralElementInstance>();
+		for (int index = 0; index < tableItems.size(); index++) {
+			TableItem item = tableItems.get(index);
+			if (!item.isDisposed() && item.getChecked()) {
+				Specification slectedspec = null;
```
Comment (JAmmermann-DLR) [inline de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/wizard/ReqIfMappingPage.java:198]: Typo `selectedSpec`

**Review by JAmmermann-DLR — CHANGES_REQUESTED** [review]
I also tested it via the UI, good job! I only found a small issue.

File: de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/wizard/ReqIfMappingPage.java (hunk 2 above)
Comment (franzTobiasDLR) [reply de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/wizard/ReqIfMappingPage.java:112]: Nice catch :+1

Comment (franzTobiasDLR) [reply de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/wizard/ReqIfMappingPage.java:112]: I'm disposing all unused editors now.

**Review by JAmmermann-DLR — APPROVED** [review]

**Review by SaMuellerDLR — APPROVED** [review]
Approved!

**Pull request merged**